app/models/spotlight/exhibit.rb

```diff
@@ -79,8 +80,8 @@ def solr_data
       end
     end
 
-    def reindex_later
-      Spotlight::ReindexJob.perform_later(self)
+    def reindex_later(user = nil)
```

Comment:
added checking in the view to properly handle nil user, and a view test for nil user:
https://github.com/projectblacklight/spotlight/pull/1662/files#diff-b2c42be6424785aafeaae4f6bfe7a094R18
https://github.com/projectblacklight/spotlight/pull/1662/files#diff-c46dbe4a4b76306e1fd38e90047cd3b5R76

can do otherwise if that's preferred, though.